docs/pages/documentation/3-querying-with-graql/define-queries.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Define Queries
+keywords: graql, query, define
+last_updated: August 10, 2016
+tags: [graql]
+summary: "Graql Define Queries"
+sidebar: documentation_sidebar
+permalink: /documentation/graql/define-queries.html
+folder: documentation
+---
+
+The page documents use of the Graql `define` query, which will define a specified [variable pattern](#variable-patterns)
+describing a schema in the knowledge base. To follow along, or experiment further, with the examples given below, please
+load the *basic-genealogy.gql* file, which can be found in the *examples* directory of the Grakn installation zip, or on
+[Github](https://github.com/graknlabs/grakn/blob/master/grakn-dist/src/examples/basic-genealogy.gql).
+
+{% include note.html content="If you are working in the Graql shell, don't forget to `commit` to store a definition in
```

Comment:
`If you are working in the Graql shell, don't forget to `commit.` 

Same reasons as above